recipes/parso/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "parso" %}
+{% set version = "0.1.1" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "5815f3fe254e5665f3c5d6f54f086c2502035cb631a91341591b5a564203cffb" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
```

Comment:
Can this be `noarch: python`?

https://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages